src/api/status/public/index.html

```diff
@@ -474,6 +474,22 @@ <h4 class="mb-0" id="total-contributors-satellite">&nbsp;</h4>
               </div>
             </div>
           </div>
+          <div class="col-xl-3 col-sm-6 mb-xl-0 mb-4 mt-4">
+            <div class="card">
+              <div class="card-header p-3 pt-2">
+                <div
+                  class="icon icon-lg icon-shape bg-gradient-success shadow-success text-center border-radius-xl mt-n4 position-absolute"
+                >
+                  <i class="material-icons opacity-10">person</i>
+                </div>
+                <div class="text-end pt-1">
+                  <p class="text-sm mb-0 text-capitalize">Total posts</p>
+                  <h4 class="mb-0" id="totalPosts"></h4>
+                </div>
+              </div>
+              <hr class="dark horizontal my-0" />
```

Comment:
This is breaking the round design:
<img width="286" alt="Screen Shot 2021-11-18 at 3 34 49 PM" src="https://user-images.githubusercontent.com/53121061/142492799-3cb9cd87-fddb-4ac0-9f8b-fa2adc4062d0.png">

You can remove this `<hr>` and add a `<div>` with `class="card-footer"`. Something like this should fix it.

Reply:
Hi I fixed. Can you check if its the same as your design ?